src/pal/src/eventprovider/lttng/eventprovdotnetruntimerundown.cpp

```diff
@@ -0,0 +1,1576 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+/******************************************************************
+
+DO NOT MODIFY. AUTOGENERATED FILE. 
+This file is generated using the script from <root>/src/inc/genXplatEtw.pl
+
+******************************************************************/
+
+
+#define TRACEPOINT_CREATE_PROBES
+#define TRACEPOINT_DEFINE
+#include "lttng/tpdotnetruntimerundown.h"
+extern "C" BOOL  EventEnabledCLRStackWalkDCStart(){ return TRUE;}
+extern "C" ULONG  FireEtwCLRStackWalkDCStart(
+                    const unsigned short  ClrInstanceID,
+                    const unsigned char  Reserved1,
+                    const unsigned char  Reserved2,
+                    const unsigned int  FrameCount,
+                    const void ** Stack
+)
```

Comment:
It is a choice of abstraction which we might revisit when 2.7 hits mainline. Currently LTTng is one of the many Logging frameworks we are targeting, so this makes sense for now